README.rst

```diff
@@ -250,6 +250,10 @@ Choose from the list of available rules:
 
   - ``allow_single_line_closure`` (``bool``): whether single line lambda notation
     should be allowed; defaults to ``false``
+  - ``position_after_control_structures`` (``'same'``): whether the opening brace
```

Comment:
I have subtracted it to separated issue: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2685
just to not forgot about it